src/bootstrap/tool.rs

```diff
@@ -33,6 +33,47 @@ struct ToolBuild {
     allow_features: &'static str,
 }
 
+fn tooling_output(build: &ToolBuild) -> String {
```

Comment:
nit: `build` usually means `bootstrap::Build`, not `tool::ToolBuild` - it would be nice to call this `tool` or something instead